atomic_reactor/util.py

```diff
@@ -752,10 +752,16 @@ def get_config_from_registry(image, registry, digest, insecure=False,
         image, registry, digest=digest, insecure=insecure,
         dockercfg_path=dockercfg_path, version=version)
     response.raise_for_status()
+    config_digest = response.json()['config']['digest']
+
+    config_response = query_registry(
+        image, registry, digest=config_digest, insecure=insecure,
+        dockercfg_path=dockercfg_path, version=version)
```

Comment:
Ah, ok. Would you mind interactively re-basing to merge that final commit together with the first one so the commit history adds logical steps in order?